tests/cmd_test_common.sh

```diff
@@ -20,24 +20,33 @@
 # Very verbose but useful for debugging.
 set -eux
 
-if [[ "$#" -ge 1 ]]; then
+# The CMake config must always be passed. It can be empty if there is
+# no multi-config.
```

Comment:
Nit: It can be empty if there is no multi-config => It is empty if the generator is not multi-config

Reply:
Done